spec/controllers/dashboard/firms_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+module Dashboard
+  RSpec.describe Dashboard::FirmsController, type: :controller do
+    let(:principal) { FactoryGirl.create(:principal) }
```

Comment:
Keeping the `FactoryGirl.` prefixes in makes it easy to pick out by eye or by search where the fixtures are being created. Inconsistency can be addressed. For me, keeping something that improves readability, even if it's not yet throughout the codebase, is a win.